modules/geo-layers/src/h3-layers/h3-hexagon-layer.js

```diff
@@ -111,35 +118,51 @@ export default class H3HexagonLayer extends CompositeLayer {
     const {
       data,
       getHexagon,
-      updateTriggers,
-
+      wireframe,
       elevationScale,
-      extruded,
       fp64,
-
+      material,
+      stroked,
+      lineWidthScale,
+      lineWidthMinPixels,
+      lineWidthMaxPixels,
+      lineJointRounded,
+      lineMiterLimit,
+      lineDashJustified,
       getColor,
       getElevation,
-      material
+      getLineColor,
+      getLineWidth,
+      getLineDashArray,
```

Comment:
Note that this layer has two modes controlled by the `highPrecision` prop. Whatever new prop you propose to add must be consistently supported by both modes, i.e. stroke support must be added to the `ColumnLayer` at some point.

While this PR may be the first one in a series of PRs, it's worth asking, do we plan to support line joint and dash arrays in the `ColumnLayer`? Even if the answer is "yes", do we have to support them right away? By adding some of these props you are committing to a large amount of work that is not asked for.

Reply:
Yes, I am planning to work on `ColumnLayer` next to add stroke support.
Noted. Will remove `line joint` and `dash arrays` props as they don't seem to be part of the proposed feature.